Podfile

```diff
@@ -14,7 +14,7 @@ pod 'UIDeviceIdentifier', '~> 0.1'
 pod 'SVProgressHUD', '~>1.1.3'
 pod 'AMPopTip', '~> 0.7'
 pod 'wpxmlrpc', '~> 0.8'
-pod 'Mixpanel', '2.5.4'
+pod 'Mixpanel', '2.7.3'
```

Comment:
I was having problems with their A/B testing UI and I thought the old SDK might be some of the issue. It seemed to improve after I updated it.